js/timers.ts

```diff
@@ -19,14 +21,9 @@ interface Timer {
 
 const timers = new Map<number, Timer>();
 
-export function initTimers() {
-  sub("timers", onMessage);
-}
-
-function onMessage(payload: Uint8Array) {
-  const msg = pb.Msg.decode(payload);
-  assert(msg.command === pb.Msg.Command.TIMER_READY);
-  const { timerReadyId, timerReadyDone } = msg;
+export function onMessage(msg: fbs.TimerReady) {
+  const timerReadyId = msg.id();
+  const timerReadyDone = msg.done();
   const timer = timers.get(timerReadyId);
   if (!timer) {
     return;
```

Comment:
I would really like to see a test that exercises this code path. Maybe something like:


```js
const timers = [];
let timeouts = 0;

for (let i = 0; i < 10; i++) {
  timers[i] = setTimeout(onTimeout, 20);
}

function onTimeout() {
  ++timeouts;
  for (let i = 1; i < 10; i++) {
    clearTimeout(timers[i]);
  }
}

assert(timeouts === 1);
```

Reply:
I will start adding more tests when unit_tests.ts lands ( #448 )
added an issue for it #497